test/ integration.test.js

```diff
@@ -0,0 +1,79 @@
+const path = require('path');
```

Comment:
Does the name of this file contain a space in the beginning? 🤔

Reply:
damn, good 🦅 👁️